applications/peripheral_sensor_node/configuration/nrf54l15pdk_nrf54l15_cpuapp/app.overlay

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/{
+	/delete-node/ leds;
+
+	leds1 {
```

Comment:
Why do we need different LEDs node definition for this application?

Reply:
GPIO P2.09 is used for the SPI_MISO signal and also for LED0.
The application uses one led.